### PR TITLE
omer.faruk/EI-966-Add_VNET_Integration_FA_Module

### PR DESCRIPTION
**Pull request by omerf-hmcts**
omer.faruk/EI-966-Add_VNET_Integration_FA_Module

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -181,3 +181,34 @@ variable "shared_dashboard" {
...
+  default     = ""
+}
+
+variable "subnet_cidr" {
+  description = "Vnet Subnet CIDR"
+  type    = list(string)
+  default     = []
+}
+
+variable "revision" {
```
Comment (varunvijayagopal) [inline variables.tf:203]: why do we need this ?

File: vars/dev/notifyatt.tfvars [hunk 2]
```diff
@@ -64,3 +64,20 @@ environment       = "dev"
...
+
+# Check to see subnets..
+# https://www.davidc.net/sites/default/subnets/subnets.html?network=10.89.123.0&mask=24&division=31.f4627231
+# | CIDR block size	| Maximum available addresses	| Maximum horizontal scale (instances) |
+# |        /28	    |      11	                    |             5                        |
+# 0 / 16 / 32 / 48 / 64 / 80 / 96 / 112 / 128 /
+subnet_cidr       = ["10.89.123.0/28"]
+
+#If you want to create revision of Apps
+revision          = "-01"
```
Comment (varunvijayagopal) [inline vars/dev/notifyatt.tfvars:83]: don't think we need this

**Review by varunvijayagopal — CHANGES_REQUESTED** [review]

File: vars/dev/notifyatt.tfvars (hunk 2 above)
Comment (omerf-hmcts) [reply vars/dev/notifyatt.tfvars]: this is removed

**Review by akozichev-hmcts — APPROVED** [review]

**Comment by omerf-hmcts** [comment]
* Add Vnet Subnet Integration

**Pull request merged**